interpreter/variable/fetch.go

```diff
@@ -223,8 +223,8 @@ func (v *FetchScopeVariables) Get(s context.Scope, name string) (value.Value, er
 		return v.ctx.EnableSSI, nil
 	}
 
-	if val := v.getFromRegex(name); val != nil {
-		return val, nil
+	if val, err := v.getFromRegex(name); val != nil {
+		return val, err
```

Comment:
I think we need specific error handling because val may be `nil` even err is nil.

```suggestion
	if val, err := v.getFromRegex(name); err != nil {
		return nil, err
	} else if val != nil {
	    return val, nil
	}
```

Reply:
Done in 7ff715c6f98273350a26a9192d5b6b91b23a770e